src/components/Navbar.astro

```diff
@@ -332,6 +332,13 @@ try {
       
       <Roles client:only />
       <!-- <div class="flashing"> -->
+      <a
+        class="navbar-item px-2 is-size-5 clickable"
+        id="analytics"
```

Comment:
Change the id of the element to research. 2 html elements in a dom cannot have the same id